tests/common.go

```diff
@@ -10,48 +10,22 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
+	"math/rand"
+	"net/http"
 	"net/url"
 	"os"
 	"os/exec"
 	"path"
 	"reflect"
+	"regexp"
+	"runtime"
 	"strconv"
 	"strings"
 	"sync"
 	"time"
-	"math/rand"
-	"net/http"
-	"regexp"
-	"runtime"
 
 	"cloud.google.com/go/storage"
-	appType "github.com/portworx/torpedo/drivers/applications/apptypes"
-	"google.golang.org/api/iterator"
-	"google.golang.org/api/option"
-	"github.com/portworx/torpedo/pkg/stats"
-	optest "github.com/libopenstorage/operator/pkg/util/test"
```

Comment:
Why are these things changing?

Reply:
I think go fmt automatically sorts imports alphabetically. First comes standard library import and then external library imports

Reply:
Goland formatted it. No other changes in this file